integration/presto/src/test/scala/org/apache/carbondata/presto/util/CarbonDataStoreCreator.scala

```diff
@@ -80,7 +80,7 @@ object CarbonDataStoreCreator {
           UUID.randomUUID().toString))
       //   val factFilePath: String = new File(dataFilePath).getCanonicalPath
       val storeDir: File = new File(absoluteTableIdentifier.getTablePath)
-      CarbonUtil.deleteFoldersAndFiles(storeDir)
+//      CarbonUtil.deleteFoldersAndFiles(storeDir)
```

Comment:
delete it if not required, same for line 81

Reply:
ok